assets/js/guides.js

```diff
@@ -101,6 +108,8 @@ $(document).ready(function () {
           selectedNavLink.parent().addClass('expanded');
         }
       }
+    } else {
+      $('.sectlevel1 li a').first().addClass('selected');
```

Comment:
So if no headings are visible... We select the first one?

Reply:
Yes, we want to have the first heading already active once we land on the page.